include/astshim/FrameSet.h

```diff
@@ -409,7 +409,10 @@ friend class Object;
     };
 
     /**
-    Remove a @ref Frame Frame from a @ref FrameSet
+    Remove a @ref Frame Frame from a @ref FrameSet.
```

Comment:
I suspect the vast majority of `@ref` tags are unnecessary, so just searching for all of them might be faster than devising a script.

I assume you left the `@ref` here for forward-compatibility with such a script?